default/buildings.txt

```diff
@@ -1575,6 +1622,8 @@ BuildingType
         Population low = [[MIN_RECOLONIZING_SIZE]]
         OwnedBy empire = Source.Owner
     ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
```

Comment:
The TODO about disabling if no possibility is ok, but the enqueue restriction here is ***not*** ok; it could very well be that someone wants to build multiple colony bases in that system, and we have no reason to stop them in general.